index.js

```diff
@@ -1,30 +1,65 @@
 const cmd = require('node-cmd')
 const fs = require('fs');
+const { sep: slash } = require('path');
 const path = require('path');
+const loaderUtils = require('loader-utils');
+
+const properName = name => name.replace(/^./, c => c.toUpperCase());
+const listify = array => array.join(', ')
+    // make a comma-separated list ending with a '&' separator
+    .replace(/(, )[^,]*$/, s => ' & ' + s.split(', ')[1]);
 
 module.exports = function (source) {
 
-    var callback = this.async();
+    const compilers = {
+        transcrypt: {
+            switches: '-b -n',
+            folder: `.${slash}__javascript__`,
+            install: 'pip install transcrypt',
+            python_version: '3.x'
+        },
+        jiphy: {
+            switches: '',
+            folder: `.${slash}`,
```

Comment:
@martim00 Just realised this should be `folder: '.'` as the `${slash}` is appended on line 55 below. Happy to submit another PR?

Reply:
Sure, you're welcome!